lib/suma/admin_api/commerce_offerings.rb

```diff
@@ -80,14 +81,24 @@ class PicklistEntity < BaseEntity
     Suma::AdminAPI::CommonEndpoints.create(
       self,
       Suma::Commerce::Offering,
-      DetailedEntity,
+      DetailedOfferingEntity,
     ) do
       params do
         requires :image, type: File
         requires(:description, type: JSON) { use :translated_text }
-        requires(:fulfillment_prompt, type: JSON) { use :translated_text }
-        requires(:fulfillment_confirmation, type: JSON) { use :translated_text }
-        requires :fulfillment_options,
+        requires :fulfillment_prompt, type: JSON do
+          requires :en, type: String, allow_blank: true
```

Comment:
Why are you copy-pasting this rather than `use :translated_text` as it was?

Reply:
Since we allow to pass `fulfillment_prompt: {en: "", es: ""}` it will, translated_text doesn't allow_blank strings, I updated translated_text to pass this option through args